### DISCUSSION TITLE
Getting Templates

### DISCUSSION THREAD

**Discussion by xenastro** [gabrieldwight/Whatsapp-Business-Cloud-Api-Net#82] [Q&A] [answered]
Hello,

Meta has an API to retrieve templates for an account, is there an API in this project to use it?

Reference:
https://developers.facebook.com/docs/whatsapp/business-management-api/message-templates/

**Comment by gabrieldwight** [comment, marked as answer]
hi,  I have not implemented the function to create and manage templates for an account. I was planning to add a generic method to pass different message components for creating template messages and I will implement the API for retrieving templates once I get time.

**Reply by xenastro** [reply]
Thank you Gabriel.

Keep up the good work, much appreciated.
